internals/secrethub/generate.go

```diff
@@ -58,9 +64,10 @@ func (cmd *GenerateSecretCommand) Register(r command.Registerer) {
 	clause.HelpLong("By default, it uses numbers (0-9), lowercase letters (a-z) and uppercase letters (A-Z) and a length of 22.")
```

Comment:
I think we can remove this long help. This is now documented in the `charset` flag.